### DISCUSSION TITLE
OpenID connect greyed out

### DISCUSSION THREAD

**Discussion by xeac** [pimcore/pimcore#17469] [Q&A / Support]
hi team,

could you please let me know what could be the reason for OpenID connect menu greyed out and not giving me possibility to edit. 
Pimcore v. 11.3. 1 Enterprise subscription.

screenshot attached.

![openid](https://github.com/user-attachments/assets/c3a2e754-e797-4650-88e2-8e5200246ec9)

second question:
when i try to install the bundle, i am getting an error: 
_Could not find a matching version of package pimcore/openid-connect. Check the package spelling, your version constraint and that the package is available in a stability which matches your minimum-stability (stable)._

thanks for the help in advance.

**Comment by markus-moser** [comment]
Do you use the `symfony-config` configuration storage in production environment? For this constellation it is on purpose that it is not editable. 
See https://pimcore.com/docs/platform/2023.3/OpenID_Connect/Configuration/#configuration-storage
https://pimcore.com/docs/pimcore/10.2/Development_Documentation/Deployment/Configuration_Environments.html#page_Production-environment-with

If you cannot find the package maybe you did not add the enterprise repo to your composer.json?
